capture/capture.go

```diff
@@ -190,18 +171,18 @@ func (l *Listener) Filter(ifi NetInterface) (filter string) {
 	if l.trackResponse {
 		dir = " "
 	}
-	filter = fmt.Sprintf("(%s%s%s)", l.Transport, dir, port)
```

Comment:
Why this "tcp" is needed here? 
We kinda trying to speed it up, by limiting to tcp protocol?